python-spec/src/somacore/ephemeral/collections.py

```diff
@@ -122,7 +122,7 @@ class Collection(  # type: ignore[misc]  # __eq__ false positive
 """The loosest possible constraint of the abstract Measurement type."""
 
 _BasicAbstractScene = scene.Scene[
-    collection.Collection[Union[data.DataFrame, data.NDArray]], base.SOMAObject
+    data.DataFrame, collection.Collection[data.NDArray], base.SOMAObject
```

Comment:
Not sure I understand this. Are we saying a Scene _could_ be a data frame OR a collection?

Reply:
There is a comment in the `scene.Scene` class definition:

```
    # This class is implemented as a mixin to be used with SOMA classes.
    # For example, a SOMA implementation would look like this:
    #
    #     # This type-ignore comment will always be needed due to limitations
    #     # of type annotations; it is (currently) expected.
    #     class Scene(  # type: ignore[type-var]
    #         ImplBaseCollection[ImplSOMAObject],
    #         somacore.Scene[
    #             Union[ImplGeometryDataFrame, ImplPointCloud], # _SpatialDF
    #             ImplImageCollection,                          # _ImageColl
    #             ImplSOMAObject,                               # _RootSO
    #         ],
    #     ):
    #         ...
```

The `_BasicAbstractScene` is just defining the inner `somacore.Scene` piece.